### ISSUE TITLE
add descriptive tooltips to Register New User window

### ISSUE THREAD

**Issue by cseppan** [USEPA-OAQPS/emf#96]
From the accessibility test report: 

> The form fields should be descriptive or have descriptive tooltip. Example: Name can be labeled as Full name, Phone format could be (000-000-0000), Password to have certain characters etc.

**Comment by cseppan** [comment]
Confirmed that field labels show up. Need to fix typo in panel labels (Uaer -> User).
